import.Rmd

```diff
@@ -65,7 +67,7 @@ Most of readr's functions are concerned with turning flat files into data frames
   [webreadr](https://github.com/Ironholds/webreadr) which is built on top 
   of `read_log()`, but provides many more helpful tools.)
 
-readr also provides a number of functions for reading files off disk into simpler data structures:
+readr also provides a number of functions for reading files from disk into data structures simpler than a data frame:
```

Comment:
Not sure whether the convention is to capitalise `readr` if it's the start of a sentence. In either case, best to be consistent.